bogoworld/main.py

```diff
@@ -0,0 +1,58 @@
+from random import randint
+import sys, time
+
+#prints a string by:
+# printing a random character
+# checking against the current desired character
+# backspacing if the result was incorrect
+#
+#Arguments:
+# s - the string to be printed (ASCII-only)
+# d - the average delay between correctly printed characters (in seconds)
+def bogoprint(s, delay=0.2):
+
+ #generator comprehension that returns the numeric value of every character in a string, one at a time
+ g = (ord(char) for char in s)
+
+ try:
+  #pre-generate the first character
+  i = next(g)
+  while 1:
+
+   #randomly generate and print a valid ASCII character
+   c = randint(32,126)
+   sys.stdout.write(chr(c))
+   sys.stdout.flush()
+
+   if i == c:
+    #if the printed character was correct, generate the next character in the sequence.
+    i = next(g)
+
+   else:
+    #print a backspace to remove the incorrect generated character
+    sys.stdout.write('\b')
+    sys.stdout.flush()
```

Comment:
Frequently, when you feel like code should be commented, it's better to create a subroutine.

For example, if this function is defined:
```python
def stdout_backspace():
    sys.stdout.write('\b')
    sys.stdout.flush()
```

You can simplify a comment to 
```suggestion
    stdout_backspace()
```